packages/create-next-app/templates/default/pages/index.js

```diff
@@ -6,7 +6,7 @@ const Home = () => (
   <div>
     <Head>
       <title>Home</title>
-      <link rel='icon' href='/static/favicon.ico' />
+      <link rel='icon' href='/favicon.ico' />
```

Comment:
I think this can just be removed at this point 🤔 `/favicon.ico` is automatically loaded in browsers.

Reply:
We should come up with a different file to show the feature behavior then.

Reply:
No I think it's fine to just drop that `<link>` tag there 👍